resources/assets/components/WithReviewing/index.js

```diff
@@ -190,6 +191,26 @@ const reviewComponent = (Component, data) => {
       }
     }
 
+    // Rotate a Post Image.
+    rotate(postId) {
+      const post = this.state.posts[postId];
+
+      let response = this.api.post(`images/${postId}?rotate=90`);
+
+      return response.then((json) => {
+        this.setState((prevState) => {
+          const newState = {...prevState};
+
+          // Add a cache-busting string to the end of the image url
+          // so that it changes and triggers a re-render.
+          newState.posts[postId].media.url = `${json.url}?time=${new Date()}`;
```

Comment:
Can this query string be removed now that it's being returned in the response?

Reply:
This ultimately hits a `image/{post_id}` endpoint that just rotates the image and returns the url we already had for it. So we don't actually update the url. But what I could do is have that return a new url with the time stamp on it, but again we aren't really updating the record in the DB. I went back and forth on whether we should do that, but since we use the model service provider to create an event record for every update, I have to be a little bit more careful that we only log meaningful events.

Reply:
Mmm gotcha. I like the cleanliness of always returning the URL from one place, but I do get the data trade-off of whether or not we want events for stuff like this.